src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -40,7 +40,7 @@ internal static class Obsoletions
         internal const string BinaryFormatterMessage = "BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.";
         internal const string BinaryFormatterDiagId = "SYSLIB0011";
 
-        internal const string CodeBaseMessage = "Use Location instead.";
+        internal const string CodeBaseMessage = "Assembly.CodeBase and Assembly.EscapedCodeBase are only included for .NET Framework compatibility. Use Assembly.Location instead.";
```

Comment:
What is this used for? It looks like attribute usages are using a literal anyway?

Reply:
The `src` projects all use this, and then the `ref` assemblies use the literal strings.  But since the `src` projects were changed in this PR, that isn't obvious.